src/components/asset-list/RecyclerAssetList2/profile-header/ProfileNameRow.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { ButtonPressAnimation } from '@/components/animations';
+import { Icon } from '@/components/icons';
+import { Box, Inline, Text, useForegroundColor } from '@/design-system';
+import { useAccountProfile, useDimensions } from '@/hooks';
+import { useNavigation } from '@/navigation';
+import { abbreviateEnsForDisplay } from '@/utils/abbreviations';
+import Routes from '@rainbow-me/routes';
+
+export const ProfileNameRowHeight = 16;
+
+export function ProfileNameRow() {
+  // ////////////////////////////////////////////////////
+  // Account
+  const { accountENS, accountName } = useAccountProfile();
+
+  // ////////////////////////////////////////////////////
+  // Name & press handler
+
+  const { navigate } = useNavigation();
+
+  const onPressName = () => {
+    navigate(Routes.CHANGE_WALLET_SHEET);
+  };
+
+  const name = accountENS
+    ? abbreviateEnsForDisplay(accountENS, 20)
+    : accountName;
+
+  // ////////////////////////////////////////////////////
+  // Colors
+
+  const iconColor = useForegroundColor('secondary60');
+
+  // ////////////////////////////////////////////////////
```

Comment:
i would discuss if this is the pattern we want to have. Maybe as a code-foundation part? We have something similar in `/bottom-sheet` but with a bit different semantic: 

![image](https://user-images.githubusercontent.com/25709300/188107688-51f03626-6083-4689-865d-c57ce18b02b6.png)

Reply:
Yeah, good idea!